lib/QubitFlatfileImport.class.php

```diff
@@ -468,6 +468,25 @@ private function handleByteOrderMark($fh)
 
   protected function handleColumnRenaming()
   {
+    // Assign names to blank columns
+    $baseLabel = 'Untitled';
+    $labelNumber = 1;
+    foreach ($this->columnNames as $index => $name)
+    {
+      if (empty($name))
+      {
+        // Increment label number if column already exists
+        while(in_array($baseLabel . $labelNumber, $this->columnNames))
```

Comment:
not sure if it's worth breaking out into a helper function since it's only several lines, but generating a new column name seems outside the scope of handleColumnRenaming?